src/sfctl/tests/mesh_test.py

```diff
@@ -0,0 +1,91 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# -----------------------------------------------------------------------------
+
+"""Custom mesh command tests"""
+
+import unittest
+import json
+import os
+import shutil
+from knack.util import CLIError
+from sfmergeutility import SFMergeUtility
+import sfctl.custom_deployment as sf_r
+from sfctl.custom_deployment import ResourceType
+
+class MeshTests(unittest.TestCase):
+    """Mesh command tests """
+
+    def get_actual_json_file(self, actual_json_files, generated_json_file):
+        """Gets the actual json file w.r.t the generated json file"""
+        resource_name = sf_r.get_resource_name(generated_json_file)
+        generated_resource_type = sf_r.get_resource_type(generated_json_file)
+        return_file = None
+        for actual_json_file in actual_json_files:
+            actual_resource_type = sf_r.get_resource_type(actual_json_file)
+            if resource_name in actual_json_file and generated_resource_type == actual_resource_type: # pylint: disable=line-too-long
+                return_file = actual_json_file
+                break
+        self.assertNotEqual(return_file, None)
+        return return_file
+
+    def test_merge_utility(self):
+        """Test merge utility if it is generating the correct jsons"""

```

Comment:
We should not be testing the merge utility here. All merge utility tests should be done in its own repo